generated/api/services/algorithm.service.ts

```diff
@@ -61,14 +61,19 @@ export class AlgorithmService extends BaseService {
   getAlgorithms$Response(params?: {
     page?: number;
     size?: number;
-
+    sort?: string;
```

Comment:
Don't do that. These files are generated and will be overwritten.

Reply:
I did not want to generate everything to test the new parameter. I will do after merging the master into our branch